gui/index.html

```diff
@@ -101,27 +101,27 @@
 
     buttonY_clicked = function () {
       axes = [0, 0]
```

Comment:
I think it would be better to send the complete joystick message instead of the fields that we are using. This would elminate the need to modify this section of the code in the future.

Axes: Array of 6
Buttons: Array of 12

Reply:
I noticed that the buttons of the modes changed. I will implement the changes and push them